source/test/F0.Minesweeper.Logic.Tests/MinefieldTest.cs

```diff
@@ -1,3 +1,4 @@
+using F0.Minesweeper.Logic.Abstractions;
```

Comment:
After the change regarding the `Location`, we can remove a now unnecessary cast to `(Location)` in `MinefieldTest.MinefieldUncover_Should_ReturnAtLeastTheClickedLocation()`.

Reply:
Aaaaand it's gone :)